moci/js/modules/network.js

```
@@ -32,101 +32,46 @@ export default class NetworkModule {
+			{ prefix: 'wireless', save: () => this.saveWireless() },
+			{ prefix: 'forward', save: () => this.saveForward() },
+			{ prefix: 'fw-rule', save: () => this.saveFirewallRule() },
+			{ prefix: 'static-lease', save: () => this.saveStaticLease() },
+			{ prefix: 'dns-entry', save: () => this.saveDnsEntry() },
+			{ prefix: 'host-entry', save: () => this.saveHostEntry() },
+			{ prefix: 'ddns', save: () => this.saveDDNS() },
+			{ prefix: 'qos-rule', save: () => this.saveQoSRule() },
+			{ prefix: 'wg-peer', save: () => this.saveWgPeer() }
+		];
+
+		modals.forEach(m =>
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)